src/ReferenceDataService/GetScreeningData/local.settings.json.template

```diff
@@ -0,0 +1,11 @@
+{
+  "IsEncrypted": false,
+  "Values": {
```

Comment:
Can you also include `"ASPNETCORE_ENVIRONMENT": "Development"` in the values, please